src/utils/createTheme.stories.js

```diff
@@ -45,48 +80,89 @@ const PrimaryButton = styled(BoxButton)`
     }
 `;
 
-const options = {
-    Default: null,
-    Blue: '#1122cc',
-    Red: '#cc1100',
-    Yellow: '#ffdd11',
-    Green: '#118811',
-    Black: '#000000',
-    White: '#ffffff',
-};
-
 const ThemeDemo = styled.div`
     width: 200px;
     min-height: 500px;
     padding: 8px;
-    margin: 5px;
     border-radius: 4px;
     display: flex;
     flex-direction: column;
     color: ${p => p.theme.primary.foreground};
 
     border: 1px solid ${p => p.theme.primary.border};
     background: ${p => p.theme.primary.background};
-    background-image: linear-gradient(
+    /* background-image: linear-gradient(
```

Comment:
do you want to leave these commented? or just remove them?

Reply:
I left the key in to avoid a breaking change but we aren't going to be using the gradient for the sidebar right now... I'll remove this from the doc